test/support/utils.h

```diff
@@ -967,11 +968,19 @@ struct __kernel_name_with_idx
 
 template <int idx, typename KernelParams>
 constexpr auto
-get_new_kernel_params(KernelParams)
+get_new_kernel_params([[maybe_unused]] KernelParams params)
 {
-    return oneapi::dpl::experimental::kt::kernel_param<
+    auto new_params = oneapi::dpl::experimental::kt::kernel_param<
```

Comment:
This local variable doesn't required in come condition check variants...

Reply:
It's been removed.